ios/RNInstabug/InstabugSessionReplayBridge.m

```diff
@@ -45,6 +47,48 @@ + (BOOL)requiresMainQueueSetup
     resolve(link);
 }
 
+- (NSArray<NSDictionary *> *)getNetworkLogsArray:
+     (NSArray<IBGSessionMetadataNetworkLogs *>*) networkLogs {
+     NSMutableArray<NSDictionary *> *networkLogsArray = [NSMutableArray array];
+    
+    for (IBGSessionMetadataNetworkLogs* log in networkLogs) {
+          NSDictionary *nLog = @{@"url": log.url, @"statusCode": @(log.statusCode), @"duration": @(log.duration)};
+          [networkLogsArray addObject:nLog];
+    }   
+    return [networkLogsArray copy];
+}
+
+- (NSDictionary *)getMetadataObjectMap:(IBGSessionMetadata *)metadataObject {
+    return @{
+        @"appVersion": metadataObject.appVersion,
+        @"OS": metadataObject.os,
+        @"device": metadataObject.device,
+        @"sessionDurationInSeconds": @(metadataObject.sessionDuration),
+        @"hasLinkToAppReview": @(metadataObject.hasLinkToAppReview),
+        @"launchType": @(metadataObject.launchType),
```

Comment:
I believe we should map this to the value on the React Native side like we did on Android, no? 👀

Reply:
I believe we are good on IOS side, constants are structured differently from android

Reply:
I assume that's because iOS doesn't map the values in constants to `key: key` right? 👀